README.md

```diff
@@ -1,2 +1,115 @@
-# example-jest.puppeteer.screenshots
-Example capturing screenshots on test failures using Jest &amp; Puppeteer
+# Example Jest Puppeteer Capturing Screenshots
+*Example repo* to show how Jest, Puppeteer, and Testspace can work together to capture screenshots of test failures. The repo demonstrates how to get `screenshots` of failing tests and attaches them to the associated `suite` when publishing test results to Testspace.
+
+There are two use cases supported:
+  1. Capture a screenshot of a test failure
+  2. Capture an image difference when a visual test fails using the [jest image snapshot](https://github.com/americanexpress/jest-image-snapshot) package
+
+When publishing Results to Testspace the following command line is used:
+
+```
+testspace junit.xml @./screenshots-list.txt
+```
+
```

Comment:
I find this paragraph unnecessary right here... may be at the end makes more sense